src/marginaldamages.jl

```diff
@@ -1,41 +0,0 @@
-include("fund.jl")
```

Comment:
Same here, lets not rename files for now, and then I'll review these once they have the same old filename.